src/components/Answer.vue

```diff
@@ -25,6 +25,9 @@ export default Vue.extend({
   props: [
     'answer',
   ],
+  data: () => ({
+    displayProbability: (process.env.VUE_APP_DISPLAY_PROBABILITIES ?? 'true') === 'true'
```

Comment:
not very used to this js syntax: `(A ?? B)` stands for something like `(A === B)` ?

Reply:
That's a quite new syntax indeed.

It means if this is null or undefined, then chose the other.

The nullish coalescing operator (what a barbarian name :)): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
ok, so `??` is more `||` but without the problematic `''` or `0` values, right ?